lib/convert.js

```diff
@@ -167,7 +169,11 @@
         }
 
         // Write an image of format <format> to STDOUT
-        args.push(format + ":-");
+        if (format === "jpg" && useFlite && settings.useOptimalJPEG) {
```

Comment:
apparently :...-fast is the same as :- and :...-default
I'll add precomputed tomorrow, although I am strongly against ever making it user facing ( which makes me wonder why to even add it)